src/containerapp/setup.py

```diff
@@ -17,7 +17,7 @@
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
 
-VERSION = '0.3.20'
+VERSION = '0.3.21'
```

Comment:
could you hold on the version upgrade here but keeping release note in history? In this way, we could release multiple features together.

Reply:
Done. Reverted the version change. Retained the history notes.